map/src/components/result.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { MarkerIdAndInfo } from 'src/components/map';
+
+import styled from '../styling';
+import MarkerType from './marker-type';
+
+interface Props {
+  result: MarkerIdAndInfo | null;
+  index: number;
+  setSelectedResult: (selectedResult: MarkerIdAndInfo | null) => void;
+}
+
+const Div = styled('div')`
```

Comment:
@jensren I tried to apply styling to the class itself like in the [`Results` component](https://github.com/reach4help/reach4help/blob/a6809bdb23c3786a5ce75af38f4858a15ea1ed09/map/src/components/results.tsx#L129-L131)  , but not sure why it didn't work here, so I used the regular styled components for styling. Could you try it once and see if it works??